src/components/Stepper/Stepper.tsx

```diff
@@ -225,6 +228,39 @@ export const Stepper: FC<StepperProps> = React.forwardRef(
       onChange?.(index, event);
     };
 
+    const scrollToSectionAndFocus = useCallback(
+      (targetSectionId: string) => {
+        if (!scrollableSection || !targetSectionId) return;
+
+        const sectionElement = document.getElementById(
+          targetSectionId
+        ) as HTMLElement;
+
+        if (!sectionElement) return;
+
+        sectionElement.scrollIntoView({ behavior: 'smooth' });
+
+        setTimeout(() => {
+          const firstFocusable = sectionElement.querySelector(
+            'input, textarea, select, [tabindex="0"]'
+          ) as HTMLElement;
```

Comment:
what is this ?? why do we still have a query selector here. please done introduce hacks